Cargo.toml

```diff
@@ -171,11 +171,31 @@ unsafe_op_in_unsafe_fn = "deny"
 
 [workspace.lints.clippy]
 all = { level = "deny", priority = -1 }
+as_ptr_cast_mut = { level = "deny" }
```

Comment:
Is there a collection we can list here instead of enumerating all of them? Why isn't this `all`?

Reply:
`all` means `all lints that are on by default (correctness, suspicious, style, complexity, perf)`

Reply:
but seems like you could just deny other groups instead of individual lints